tests/validation/tests/v3_api/test_catalog_library.py

```diff
@@ -24,6 +24,9 @@
 from .common import USER_TOKEN
 from .common import get_user_client
 
+# Skipping all the tests from this module because the library_catalog charts are not supported on latest
```

Comment:
Do we need to skip if Rancher 2.5.x @sowmyav27 or should we focus on 2.6.x only this is mainly for 2.6 failures due to support issues.

Reply:
Why are we skipping this @rishabhmsra ?

Reply:
@sowmyav27, since `test_catalog_library` tests are all related to v1 charts validation, it's failing because of the compatibility. I've DM'ed you the jenkins build logs for more details.

Reply:
IMO we should add a check to skip these tests only if the rancher versions are v2.6 and greater.

Reply:
Checked the convo. sounds good. Thanks Rishabh.